### ISSUE TITLE
Add reparameterisation for logNormal

### ISSUE THREAD

**Issue by null-a** [probmods/webppl#918]
PR #911 added a log normal distribution but it doesn't yet support reparameterisation.

**Comment by zemlyansky** [comment]
@null-a, we could use the standard normal distribution as a base function, then transform it to log-normal scaling and shifting it to parametrized normal with latter exponentiation

```javascript
base: function() {
  return new Gaussian({mu: 0, sigma: 1});
},
transform: function(x) {
  'use ad';
  return Math.exp(this.params.sigma * x + this.params.mu);
}
```
Is it a correct approach?

**Comment by null-a** [comment]
> Is it a correct approach?

Yes, I think that will do it!

**Comment by null-a** [comment]
Closed by #930.
